jest.config.js

```diff
@@ -25,8 +25,8 @@ module.exports = {
   ],
   coverageThreshold: {
     global: {
-      statements: 43,
-      branches: 39,
+      statements: 42,
```

Comment:
I reduced the coverage thresholds because the component tests for the reading list no longer exist. I've converted them to E2E tests. @djuber, this is something I haven't had time to look into/do yet, but it'd be good to aggregate the test coverage from jest with that of Cypress so that we can get a true picture of the code coverage for the front-end.